BUILDING.md

```diff
@@ -32,8 +32,13 @@ HBase 2 and Hadoop 3
 Phoenix 5.x requires Hadoop 3. While HBase 2.x is compatible with Hadoop 3, the public Maven HBase
 artifacts are built with Hadoop 2, and are not.
 
-For this reason, when building Phoenix, you need to rebuild
-HBase with Hadoop 3, and install it to the local maven repo of the build host.
+With the 2.5.2 release HBase has started releasing Maven artifacts built with Hadoop3.
+Where available, use the Maven artifacts with the `-hadoop3` postfix in the version,
+i.e. `2.5.2-hadoop3`, and ignore the rest of this section.
+For Hbase 2.5, Phoenix already uses the -hadoop3 version by default.
```

Comment:
tiny nit: the B in HBase is capitalized.

Reply:
Thanks, will fix before commit.